pandas/core/ops.py

```diff
@@ -637,15 +632,9 @@ def safe_na_op(lvalues, rvalues):
             with np.errstate(all='ignore'):
                 return na_op(lvalues, rvalues)
         except Exception:
-            if isinstance(rvalues, ABCSeries):
```

Comment:
`isinstance(rvalues, ABCSeries)` is always `False` because before calling `safe_na_op` we do this same check and if `True` set `rvalues = rvalues.values`.  If reviewers want to call this out-of-scope I'll remove it and make a separate PR.